### PR TITLE
Fixing alias length

### PR DESCRIPTION
**Pull request by Demmonius**

### CODE REVIEW

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit 389c2db03b0d4a8509c17b48b94055db1073fcf8 successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-oxbiazp2i.now.sh
	Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deploy-alias-

**Comment by Vadorequest** [comment]
[GitHub Actions]
E2E tests SUCCESS

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit 50bfb7d5e40105ac9183f7e86249c892e9df1dca successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-1tcfcsq2g.now.sh
	Deployment aliased as https://v42-backend-refactoring.now.sh

**Comment by Vadorequest** [comment]
[GitHub Actions]
E2E tests SUCCESS

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit 3d6cf600e7021cd1abad9659b670880d76475eb9 successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-boau1g40j.now.sh
	Deployment aliased as https://v42-azertyuiopqsdfghjklmwxcvbnazertyuiopqsdfghjklmwxc

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit e1ea53c6682d660e85a1be13297ff5097166f298 successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-oxbiazp2i.now.sh
	Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deploy-alias-

**Comment by Demmonius** [comment]
@Vadorequest the refactor about alias is now working. As you can see on the previous messages, if the branch name starts with "vX-" the alias changed and if it's too long, it only takes the first 53 characters.

**Comment by Vadorequest** [comment]
[GitHub Actions]
E2E tests SUCCESS

**Comment by Vadorequest** [comment]
>[GitHub Actions]
Deployment SUCCESS
Commit e1ea53c successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-oxbiazp2i.now.sh
Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deploy-alias-

@Demmonius The deployment link isn't valid, it doesn't end with .now.sh

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit e021f80a4d506d1ce0c6f4f30fb13aafb7270d3e successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-to7qp2m34.now.sh
	Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deplo.now.sh

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit f4a7e1b7cd66ad6c2388b2eae2a19bb63f05fa64 successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-7fwbe8jga.now.sh
	Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deplo.now.sh

**Comment by Demmonius** [comment]
@Vadorequest right, it's fix now!

**Comment by Vadorequest** [comment]
[GitHub Actions]
E2E tests SUCCESS

File: .github/workflows/deploy-zeit-staging.yml [hunk 1]
```diff
@@ -41,10 +41,15 @@ jobs:
           ZEIT_DEPLOYMENT_URL=`echo $ZEIT_DEPLOYMENT_OUTPUT | egrep -o 'https?://[^ ]+.now.sh'`
           echo "::set-env name=ZEIT_DEPLOYMENT_URL::$ZEIT_DEPLOYMENT_URL"
 
-          ZEIT_DEPLOYMENT_ALIAS=$(cat now.json | jq -r '.name')-${CURRENT_BRANCH##*/}.now.sh
+          if [[ ${CURRENT_BRANCH##*/} =~ ^v[0-9]{1,}- ]]; then # Checking if pattern matches with "vX-" where X is a number
+            ZEIT_DEPLOYMENT_ALIAS=${CURRENT_BRANCH##*/}
+          else
+            ZEIT_DEPLOYMENT_ALIAS=$(cat now.json | jq -r '.name')-${CURRENT_BRANCH##*/}
+          fi
+          ZEIT_DEPLOYMENT_ALIAS=$(echo $ZEIT_DEPLOYMENT_ALIAS | head -c 45).now.sh # Zeit alias only allow 45 characters, so we are taking the first 53 characters of this alias
```
Comment (Vadorequest) [inline .github/workflows/deploy-zeit-staging.yml:49]: @Demmonius 45 vs  53? (doc issue)

Comment (Demmonius) [reply .github/workflows/deploy-zeit-staging.yml:49]: Yes I should explain it

Comment (Vadorequest) [reply .github/workflows/deploy-zeit-staging.yml:49]: Yes please, I don't really understand it, as it

**Comment by Vadorequest** [comment]
[GitHub Actions]
Deployment SUCCESS
	 Commit 60bc445a3c588ad0e65b09d6e9acb9ebf824a481 successfully deployed to https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-5tet9ojcb.now.sh
	Deployment aliased as https://nrn-v1-ssr-mst-aptd-gcms-lcz-sty-c1-fix-deploy.now.sh

**Comment by Vadorequest** [comment]
[GitHub Actions]
E2E tests SUCCESS

**Comment by Vadorequest** [comment]
@Demmonius I'm merging, I think it's good now.

**Pull request merged**